### ISSUE TITLE
Enable audio is checked when the program starts, but is unchecked when a file is selected

### ISSUE THREAD

**Issue by nixxquality** [nixxquality/WebMConverter#118]
```c#
        private void DefaultSettings()
        {
            boxAudio.Checked =
                false;
        }
```

https://github.com/nixxquality/WebMConverter/blob/f5ab8620110795a675ffbad86f687ae07ad99353/MainForm.cs#L917-L948

**Comment by nixxquality** [comment]
ping #114

**Comment by Kyokou** [comment]
Oh - good catch! I would have caught this tonight doing my next batch of WebM's. I'll put those off until the next release. :+1:

**Comment by nixxquality** [comment]
Included in [v2.16.8](https://github.com/nixxquality/WebMConverter/releases/tag/v2.16.8).
